### PR TITLE
chore: add cri and cni to playbooks/site.yml

### PR DESCRIPTION
**Pull request by ricolin**

### CODE REVIEW

**Comment by ricolin** [comment]
relate to https://github.com/vexxhost/atmosphere/issues/711

File: playbooks/site.yml [hunk 1]
```diff
@@ -19,3 +19,19 @@
...
+      tags:
+        - cri-tools
+
+- name: Install CNI plugin
+  hosts: "{{ target | default('all') }}"
+  become: true
+  roles:
+    - role: cni_plugins
+      tags:
+        - cni-plugins
```
Comment (mnaser) [inline playbooks/site.yml:37]: @ricolin shouldn't these technically be role dependencies instead?

**Review by mnaser — CHANGES_REQUESTED** [review]

File: playbooks/site.yml (hunk 1 above)
Comment (ricolin) [reply playbooks/site.yml:37]: you mean role dependencies in vexxhost.kubernetes?

**Comment by ricolin** [comment]
I was adding this cause I thought we need this in vexxhost.kubernetes, but appears we don't. I will close this as this doesn't sounds needed

**Pull request closed without merging**